build.gradle

```diff
@@ -32,6 +32,7 @@ buildscript {
         jjwt_version = '0.11.5'
         guava_version = '32.1.2-jre'
         jaxb_version = '2.3.8'
+        checkstyle_version = '10.12.4'
```

Comment:
Nit: Why create a variable, its only used once?

---

Whenever I say `Nit` this is a take it or leave it comment.  You don't need to make a change for these types of suggestions if you don't want to and you can resolve this comment thread without a response if you so desire.  Also you can write a detailed response and make changes.

Nit comments are personal preference based and while they might be useful to consider/adopt they shouldn't block productivity.

Reply:
Good point. 
I adopted this practice long time ago, so I will reconsider it. Thanks a lot @peternied for the feedback.